app/javascript/components/ArticleForm.js

```diff
@@ -1,28 +1,60 @@
 import axios from 'axios';
-import React, { useState } from 'react';
-import { Form, FormGroup, TextInput, Modal, ModalVariant, Button } from '@patternfly/react-core';
+import React, { useState, useEffect } from 'react';
+import { Form, FormGroup, TextInput, Modal, ModalVariant } from '@patternfly/react-core';
 import PropTypes from 'prop-types';
+import { Dropdown, DropdownToggle, DropdownGroup, DropdownItem } from '@patternfly/react-core';
+import PfButton from './pfButton';
 
 const ArticleForm = ({ isOpen, handleModalToggle }) => {
   const [data, setData] = useState({
     title: '',
     body: '',
     creator: ''
   });
+  const [isDropdownOpen, setIsDropdownOpen] = React.useState(false);
+  const [users, setUsers] = useState([]);
 
+  useEffect(() => {
+    axios
+      .get('/users')
+      .then((response) => {
+        setUsers(response.data);
+        console.log(users);
+      })
+      .catch((response) => console.log(response));
+  }, [users.length]);
+  const [selectedUser, setSelectedUser] = useState('');
   const handleChange = (value, key) => {
     setData({
       ...data,
       [key]: value
     });
   };
+  const onToggle = (isDropdownOpen) => {
+    setIsDropdownOpen(isDropdownOpen);
+  };
+
+  const onSelect = (event) => {
+    const value = event.target.innerHTML;
+    setSelectedUser(value);
```

Comment:
instead of passing the string value of the user name,
can you pass the ID of the user? and also link in the database the `Article.creator` to `User`?